homework-submission/my-project/src/Header.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+class Header extends Component {
+  render() {
+    return (
+      <div>
+        <header>Todo List</header>
+      </div>
```

Comment:
it is work fine here but you don't need a < div > container because you are returning just one element you need a container just if you return more than one element